x-pack/plugins/lens/server/routes/field_stats.ts

```diff
@@ -79,13 +78,14 @@ export async function initFieldsRoute(setup: CoreSetup<PluginStartContract>) {
           },
         };
 
-        const search = async (aggs: unknown) => {
+        const search = async (aggs: Record<string, estypes.AggregationContainer>) => {
           const { body: result } = await requestClient.search({
             index: indexPattern.title,
             track_total_hits: true,
             body: {
               query,
               aggs,
+              // @ts-expect-error @elastic/elasticsearch SearchRequest doesn't declare runtime_mappings property
```

Comment:
@delvedor @mshustov Another error here

Reply:
fixed this, found another error `StoredScript.language is required`. it's likely to be fixed in a follow-up PR.